src/ui/components/Icon/styles.scss

```diff
@@ -76,3 +80,11 @@
   min-width: 46.3px;
```

Comment:
I'm guessing  it's a holdover from the whole "divide things by 3" issue with sketch files. Also maybe not but I didn't write that line 😉

I'll have a look though; `46.5` might be okay.